src/postgres-mcp-server/awslabs/postgres_mcp_server/__init__.py

```diff
@@ -12,6 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""awslabs.postgres-mcp-server"""
+"""PostgreSQL MCP Server package."""
```

Comment:
Keep as the package for consistency. Thanks!

```suggestion
"""awslabs.postgres-mcp-server"""
```